MagicalCryptoWallet/Models/Height.cs

```diff
@@ -43,15 +71,49 @@ public Height(string heightOrHeightType)
 			else this = new Height(int.Parse(trimmed));
 		}
 
+		/// <summary>
+		/// Creates and initializes a new Height instance
+		/// </summary>
+		/// <param name="type">Height type for the created instance.</param>
+		/// <exception href="NotSupportedException">When type is equal to HeightType.Chain.</exception>
 		public Height(HeightType type)
 		{
 			if (type == HeightType.Chain) throw new NotSupportedException($"For {type} height must be specified");
 			Type = type;
-			if (Type == HeightType.MemPool)
-				_value = int.MaxValue - 1;
-			else _value = int.MaxValue; // HeightType.Unknown
+			_value = (Type == HeightType.MemPool)
+				? MemPool.Value
```

Comment:
Nice PR, before merge, could you explain this? I'm a bit confused why it works, if it works:

So it calls the static `Height.MemPool` here and takes its value and runs with it. But `Height.MemPool` is this: `MemPool = new Height(HeightType.MemPool);`, which calls this same function. Shouldn't it end up in infinite cycle?

Reply:
Before this PR `MemPool` was an expression-bodied **function**, that means we got a new Height instance every time we invoke it. 

In this PR `MemPool` is an static field, that means it is initialized only once before its first usage so, there is no infinite cycle possible.